square/square.go

```diff
@@ -0,0 +1,25 @@
+package square
+
+// Square ...
+type Square struct {
+	StartPoint Point
+	Side       uint
+}
+
+// End ...
+func (s Square) End() Point {
+	return Point{
+		X: s.StartPoint.X + int(s.Side),
+		Y: s.StartPoint.Y + int(s.Side),
```

Comment:
I see an error in computing y

Reply:
Can u explain where is an error?
` s := square.Square{
		StartPoint: square.Point{X: 1, Y: 1},
		Side:       150,
	}
	fmt.Println(s.End())
`

It works fine and returns:
` {151 151} `